pkgs/development/python-modules/django_tagging/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, buildPythonPackage, fetchurl, django }:
+
+buildPythonPackage rec {
+  name = "django-tagging-0.4.5";
+
+  src = fetchurl {
```

Comment:
I recommend making several changes at the same time. Most importantly is using

a. `fetchPypi` when possible
b. have `pname`, `version` and `name` attributes so we can more easily update packages

Example https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/jupyter_client/default.nix